README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/datasciencebr/whistleblower.svg?branch=master)](https://travis-ci.org/datasciencebr/whistleblower)
-
 [![Code Health](https://landscape.io/github/datasciencebr/whistleblower/master/landscape.svg?style=flat)](https://landscape.io/github/datasciencebr/whistleblower/master)
+[![Coverage Status](https://coveralls.io/repos/github/gusttavoaguiarr/whistleblower/badge.svg?branch=coverage)](https://coveralls.io/github/gusttavoaguiarr/whistleblower?branch=coverage)
```

Comment:
This line is linking to the badge and report from `gusttavoaguiarr`'s account, not `datasciencebr`'s account.